packages/url_launcher/url_launcher_platform_interface/pubspec.yaml

```diff
@@ -3,20 +3,24 @@ description: A common platform interface for the url_launcher plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.0.8
+version: 2.0.0-nullsafety
```

Comment:
If we change the public api to return non-nullable bool, then it is probably just a minor version change? cc @blasten for versioning.